server/models/Author.js

```diff
@@ -0,0 +1,20 @@
+const { Model } = require("objection");
+
+class Author extends Model {
+  static get tableName() {
+    return "authors";
+  }
+
+  static async create(val) {
```

Comment:
This method is not needed. Let's use the create method directly in the controller. Look at the user registration as an example.